app/views/embeddable/open_responses/_form.html.haml

```diff
@@ -15,9 +15,10 @@
       .ul
         = render :partial => 'shared/edit_prediction', :locals => { :f => f }
 
-    :javascript
-      $(function() {
-          $('.wysiwyg').wysiwyg({"controls": { "html": { "visible": true }, "h1": { "visible": false }, "h2": { "visible": false }, "h3": { "visible": false } } });
-      });
   = f.submit "Cancel", :class => 'close'
   = f.submit "Save", :class => 'embeddable-save'
+
+  :javascript
+    $(function() {
+      initTinyMCE('.wysiwyg', window.TinyMCEConfigMinimal);
```

Comment:
It looks like this would apply the minimal config to all `.wysiwyg` elements on the page. What happens when there are some elements that should be minimal and some that should be basic?  Or are there never both in the DOM at the same time?